charts/datasaur/values.yaml

```diff
@@ -23,6 +23,7 @@ services:
       targetMemoryUtilizationPercentage: 80
   worker:
     critical:
+      enabled: true
```

Comment:
have you update the setting for nginx's max file upload size?

Reply:
we have variable for this
`ingress.proxyproxyBodySize`